src/components/Pagination/Pagination.tsx

```diff
@@ -1,24 +1,23 @@
-import { useNavigate } from 'react-router-dom';
-import { usePageContext } from '../../context/PageContext/PageContext';
+import { useSearchParams } from 'react-router-dom';
 
 type PropsType = {
-  nextPage: string | null;
-  prevPage: string | null;
+  nextPage: string | null | undefined;
+  prevPage: string | null | undefined;
```

Comment:
if you have to wait for undefined to come, then you are doing something wrong, you need to figure out why it comes and eliminate it